src/js/StyledIcon.js

```diff
@@ -43,10 +45,13 @@ const colorCss = css`
   }
 `;
 
-const IconInner = ({
+const IconInner = forwardRef(
+  ({
  a11yTitle, color, size, theme, ...rest
-}) => (
-  <svg aria-label={a11yTitle} {...rest} />
+}, ref) => {
+    const iconRef = useForwardedRef(ref);
```

Comment:
I don't think we need `iconRef` and `userForwardedRef()`. I think we can just pass `ref` directly to the `<svg />`.

Reply:
Looks good! Implemented now

Reply:
I think you still need to re-generate the icon component files to use the new StyledIcon.